include/deal.II/lac/petsc_compatibility.h

```diff
@@ -53,7 +53,7 @@ namespace PETScWrappers
 #if DEAL_II_PETSC_VERSION_LT(3, 7, 0)
     const PetscErrorCode ierr = PetscOptionsSetValue (name.c_str (), value.c_str ());
 #else
-    const PetscErrorCode ierr = PetscOptionsSetValue (NULL, name.c_str (), value.c_str ());
+    const PetscErrorCode ierr = PetscOptionsSetValue ((nullptr), name.c_str (), value.c_str ());
```

Comment:
I have called SLEPc functions with `nullptr` and things work correctly, so this is fine (aside from the extra parentheses).